transform/mattermost-analytics/models/marts/product/dim_server_info.sql

```diff
@@ -59,7 +45,10 @@ select
     si.first_count_registered_active_users,
     si.last_count_registered_active_users,
     si.last_daily_active_users,
-    si.last_monthly_active_users
+    si.last_monthly_active_users,
+    si.last_server_ip,
+    coalesce(ip.last_known_ip_country, 'Unknown') as last_known_ip_country
```

Comment:
can this (ever) happen given the similar code in int_server_ip_to_country.sql?

Reply:
Shouldn't be needed, but added it as an extra safeguard.